src/language/dafnyInstallation.ts

```diff
@@ -89,15 +89,63 @@ export async function getCompilerRuntimePath(context: ExtensionContext): Promise
   return configuredPath;
 }
 
+// We cache the language server runtime path so that we don't need to copy it every time.
+let LanguageServerRuntimePath: string | null = null;
+
+function isNullOrEmpty(s: string | null): boolean {
+  return s === '' || s == null;
+}
+
 export async function getLanguageServerRuntimePath(context: ExtensionContext): Promise<string> {
-  const configuredPath = await ifNullOrEmpty(
-    getConfiguredLanguageServerRuntimePath(),
+  if(LanguageServerRuntimePath != null) {
+    return LanguageServerRuntimePath;
+  }
+  const configuredLanguageServerPath = getConfiguredLanguageServerRuntimePath();
+  const mightBeRecomputed: boolean = !isNullOrEmpty(configuredLanguageServerPath);
```

Comment:
nit: The type-specification is unnecessary here.

```suggestion
  const mightBeRecomputed = !isNullOrEmpty(configuredLanguageServerPath);
```